### ISSUE TITLE
Vulnerabilities on the package "marked"

### ISSUE THREAD

**Issue by codespede** [TypeStrong/typedoc#1847]
The current version of typedoc comes with version 3.x.x of "marked" and causes the below vulnerabilities:
```
GHSA-rrrm-qjm4-v8hf
high severity
Vulnerable versions: < 4.0.10
Patched version: 4.0.10

GHSA-5v2h-r2cx-5xgj
high severity
Vulnerable versions: < 4.0.10
Patched version: 4.0.10
```
Below are the Remediations I got from Gtihub:
```
2 marked vulnerabilities found in package-lock.json 

Remediation
Upgrade marked to version 4.0.10 or later. For example:

"dependencies": {
  "marked": ">=4.0.10"
}
or…
"devDependencies": {
  "marked": ">=4.0.10"
}
```

Hope you will take care of this soon.

**Comment by Gerrit0** [comment]
#1844

**Comment by Dayday10** [comment]
Fix

**Comment by Gerrit0** [comment]
Resolved with #1851
